backend/src/main/java/com/zpi/backend/email_type/EmailType.java

```diff
@@ -7,6 +7,19 @@
 @Data
 @Table(name = "email_types")
 public class EmailType {
+
```

Comment:
maybe add an enum type?

Reply:
i see that adding enums isn't as easy as i thought (because we have strings in db) so you can ignore these

Reply:
or maybe not, it requires something like 
Roles.valueOf(user.getRole().getName()).equals(Roles.user) in if statements